gourmet/plugins/import_export/website_import_plugins/cooksillustrated.py

```diff
@@ -1,64 +0,0 @@
-from selenium import webdriver
```

Comment:
Could you quickly say why this file is deleted?

Reply:
This seemed to be an unused file used for testing some stuff in the past. The real plugin is located in `cooksillustrated_plugin.py` and does not use this file, so I assumed it should be safe to delete it.